docs/root/faq/api/envoy_upgrade_v3.rst

```diff
@@ -0,0 +1,16 @@
+If I upgrade to Envoy 1.13+, do I need to use the v3 API?
+=========================================================
+
+The v2 API is deprecated in the 1.13.0 release (January 2020). It will be fully supported for the
+duration of 2020 and then all support for v2 will be removed from Envoy at EOY 2020.
+
+All existing v2 boostrap and xDS configuration should continue to work seamlessly in 1.13.0 and for the
+duration of 2020. Envoy internally operates at v3+, but does so by transparently upgrading
+configuration from v2 at ingestion time.
+
+Since EOQ1 2020, we have freeze the v2 API and no new features will be added. To consume these
```

Comment:
should we expect v3 to be frozen EOQ1 2021? or was this year different due to introducing the new process

Reply:
I think we'll maintain this grace period, since it gives folks who want to be tracking latest API features a quarter of supported v4 to upgrade.

Reply:
Let's move this discussion to https://github.com/envoyproxy/envoy/issues/10852, I think we may not have strict Q1 potentially.